admin/sql/updates/20200630-mbs-10921-unlist-unrelated-edits.sql

```diff
@@ -0,0 +1,286 @@
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'link', 'type', 'entity0_type') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'link', 'type', 'entity1_type') = 'work'
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'entity0', 'id')::int != er.recording
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'entity0', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    LIMIT 150 -- of 11681
+)
+RETURNING *;
+
+\echo MBS-10921 (8/12): Unlist “Reorder relationships” edits from recordings’ history...
+DELETE FROM edit_recording
+WHERE (edit, recording) IN (
+    SELECT er.edit, er.recording
+    FROM edit_recording er
+    JOIN edit e ON er.edit = e.id
+    JOIN edit_data ed ON er.edit = ed.edit
+    WHERE er.edit >= 70658947 AND er.edit < 71006471
+    AND e.type = 99
+    AND jsonb_extract_path_text(ed.data, 'link_type', 'entity0_type') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'link_type', 'entity1_type') = 'work'
+    AND jsonb_extract_path_text(data, 'relationship_order', '0', 'relationship', 'entity1', 'id')::int != er.recording
+    AND jsonb_extract_path_text(data, 'relationship_order', '0', 'relationship', 'entity1', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    LIMIT 50 -- of 3491
+)
+RETURNING *;
+
+-- III) Releases
+
+\echo MBS-10921 (9/12): Unlist “Add relationship” edits from releases’ history...
+DELETE FROM edit_release
+WHERE (edit, release) IN (
+    SELECT er.edit, er.release
+    FROM edit_release er
+    JOIN edit e ON er.edit = e.id
+    JOIN edit_data ed ON e.id = ed.edit
+    WHERE e.id >= 70658947 AND e.id < 71006471
+    AND e.type = 90
+    AND jsonb_extract_path_text(ed.data, 'type0') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'type1') = 'work'
+    AND jsonb_extract_path_text(ed.data, 'entity0', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    AND jsonb_extract_path_text(ed.data, 'entity0', 'id')::int NOT IN (
+        SELECT track.recording
+        FROM track
+        JOIN medium ON track.medium = medium.id
+        WHERE medium.release = er.release
+    )
+    LIMIT 10000 -- of 1070983
+)
+RETURNING *;
+
+\echo MBS-10921 (10/12): Unlist “Edit relationship” edits from releases’ history...
+DELETE FROM edit_release
+WHERE (edit, release) IN (
+    SELECT er.edit, er.release
+    FROM edit_release er
+    JOIN edit e ON er.edit = e.id
+    JOIN edit_data ed ON e.id = ed.edit
+    WHERE e.id >= 70658947 AND e.id < 71006471
+    AND e.type = 91
+    AND jsonb_extract_path_text(ed.data, 'type0') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'type1') = 'work'
+    AND jsonb_extract_path_text(ed.data, 'link', 'entity0', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    AND jsonb_extract_path_text(ed.data, 'link', 'entity0', 'id')::int NOT IN (
+        SELECT track.recording
+        FROM track
+        JOIN medium ON track.medium = medium.id
+        WHERE medium.release = er.release
+    )
```

Comment:
Same here re `{new,entity0,id}`.

Reply:
Added in https://github.com/metabrainz/musicbrainz-server/pull/1581/commits/04969b79a9ea8489783c7bd7090e73578593917e

Reply:
That is actually not needed either, see https://github.com/metabrainz/musicbrainz-server/pull/1581/commits/29074454c43a5123ee86f3a71d75594f5ac0fdd5.